pkg/apis/clusteroperator/validation/clusterversion.go

```diff
@@ -46,6 +47,24 @@ var validDeploymentTypeValues = func() []string {
 var versionFmt = "^v?\\d+\\.\\d+(\\..+)?$"
 var versionRegex = regexp.MustCompile(versionFmt)
 
+// validPullPolicies is a map containing an entry for every valid pull policy value.
+var validPullPolicies = map[kapi.PullPolicy]bool{
+	kapi.PullAlways:       true,
+	kapi.PullNever:        true,
+	kapi.PullIfNotPresent: true,
+}
+
+// validPullPolicyValues is an array of every valid pull policy value.
+var validPullPolicyValues = func() []string {
+	validValues := make([]string, len(validPullPolicies))
+	i := 0
+	for dt := range validPullPolicies {
```

Comment:
Nit: "for i, dt" would work here I think.

Reply:
Oh man. I copied that code over from the `validDeploymentTypeValues` function without giving it more than a glance. I wonder who wrote that code. ;)

Reply:
Some golang newb no doubt.

Reply:
That's right. `for i, dt` does not work because `validPullPolicies` is a map not an array.

Reply:
Ahh I see now.